app/.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
There's a `.eslintrc` file as well already in the repo, which just has the settings from `civic`. Does this supplement or replace it?

Reply:
Looks like it replaces it. That could be fine, but in that case the other file should be removed.

Reply:
I didn't see the `.eslintrc` in the repo! I'll use that file instead to keep everything consistent.